raiden-dapp/package.json

```diff
@@ -6,14 +6,14 @@
   "author": "brainbot labs est.",
   "scripts": {
     "serve": "vue-cli-service serve",
-    "serve:no-logging": "export NODE_ENV=nologging && vue-cli-service serve",
     "build": "vue-cli-service build",
     "test:unit": "vue-cli-service test:unit",
-    "test:e2e": "vue-cli-service test:e2e",
+    "test:e2e": "nyc vue-cli-service test:e2e --mode e2e",
```

Comment:
There is a package called [start-server-and-test](https://www.npmjs.com/package/start-server-and-test) which starts the server and as soon as it gets a 200 OK from the URL also starts the test. Might make sense to use this for headless

Reply:
Yes, I am aware of it. But here it would only make sense for running tests locally right? 
instead of starting the image by hand (using the script) we could have the package start in instead. Is that what you had in mind?

In CI it doesn't matter because cypress will run inside the container itself.